app/src/androidTest/java/com/pileproject/drive/execution/NxtCarControllerTest.java

```diff
@@ -181,16 +186,16 @@ public void whenMotorSpeedsAreNotInitialized_thenMovesForwardWithDefaultValue()
 
         controller.moveForward();
 
-        verify(leftMotor).setSpeed(NxtController.INIT_MOTOR_POWER);
-        verify(rightMotor).setSpeed(NxtController.INIT_MOTOR_POWER);
+        verify(leftMotor).setSpeed(CarControllerBase.MotorProperty.INIT_MOTOR_POWER);
```

Comment:
then why do you use that of nxt for the right motor in the below line?
am i seeing some old version?